docs/guides/run-jobs-session.mdx

```diff
@@ -31,6 +31,10 @@ service = QiskitRuntimeService()
 You can open a runtime session by using the context manager `with Session(...)` or by initializing the `Session`
 class. When you start a session,  you must specify a QPU by passing a `backend` object. The session starts when its first job begins execution.
 
+<Admonition type="note">
+If you open a session but do not submit any jobs to it for 30 minutes, the session will automatically close.
```

Comment:
What does "close" mean? Deactivated or closed permanently? I've never heard of this.

Reply:
@francabrera I had a request from you to add this info - but I can't find the context - can you help?

Reply:
this only applies to the IBM Quantum channel today. The content is right.